src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/SinkIslandCheck.java

```diff
@@ -251,6 +250,32 @@ private boolean endOrStartNodeHasAmenityTypeToExclude(final AtlasObject object)
                 || Validators.isOfType(edge.start(), AmenityTag.class, AmenityTag.PARKING_ENTRANCE);
     }
 
+    /**
+     * Finds the car accessibility value for an {@link Edge} by returning the first value found,
+     * using a descending list of tags to check. If no values are found fort the list of tags the
+     * value is assumed to be YES.
```

Comment:
The default value is passed in. Seems like the default should be "yes" like the comment says instead of passing it in. Are there times you would want the default to be something else?

Reply:
I forgot to update this comment when I added the default value argument. I ended up making the default value variable because it is YES for cars and NO for ferries.